packages/lesswrong/lib/routes.js

```diff
@@ -4,6 +4,8 @@ import { addRoute, getSetting} from 'meteor/vulcan:core';
 addRoute([
   {name:'posts.daily',      path:'daily',                 componentName: 'PostsDaily', title: "Posts by Day" },
   {name:'users.single',     path:'users/:slug',           componentName: 'UsersSingle'},
+  {name:'users.single.u',   path:'user/:slug',            componentName: 'UsersSingle'},
```

Comment:
I think these are backwards?

Reply:
Yep. They don't actually do anything (except maybe show up in debug messages occasionally), though. Will swap them back anyways though.